astroquery/mpc/tests/data/mpc_obs.dat

```diff
@@ -0,0 +1 @@
 49.8          17.3 Roj2263699"},{"designation":null,"discovery_flag":null,"number":12893,"object_type":"M","observation_date":"2003 11 24.24327","observatory_code":"699","original_record":"12893         C2003 11 24.24327 04 15 16.45 +18 06 46.6                oj2263699"},{"designation":null,"discovery_flag":null,"number":12893,"object_type":"M","observation_date":"2003 11 24.26245","observatory_code":"699","original_record":"12893         C2003 11 24.26245 04 15 15.35 +18 06 43.3                oj2263699"},{"designation":null,"discovery_flag":null,"number":12893,"object_type":"M","observation_date":"2003 11 24.28153","observatory_code":"699","original_record":"12893         C2003 11 24.28153 04 15 14.26 +18 06 40.1                oj2263699"},{"designation":null,"discovery_flag":null,"number":12893,"object_type":"M","observation_date":"2003 12 01.25005","observatory_code":"704","original_record":"12893         C2003 12 01.25005 04 08 47.74 +17 47 03.2          17.6  cj3051704"},{"designation":null,"discovery_flag":null,"number":12893,"object_type":"M","observation_date":"2003 12 01.26267","observatory_code":"704","original_record":"12893         C2003 12 01.26267 04 08 47.00 +17 47 01.1          18.2  cj3051704"},{"designation":null,"discovery_flag":null,"number":12893,"object_type":"M","observation_date":"2003 12 01.27468","observatory_code":"704","original_record":"12893         C2003 12 01.27468 04 08 46.33 +17 46 59.7          17.8  cj3051704"},{"designation":null,"discovery_flag":null,"number":12893,"object_type":"M","observation_date":"2003 12 01.28777","observatory_code":"704","original_record":"12893         C2003 12 01.28777 04 08 45.61 +17 46 57.3          18.1  cj3051704"},{"designation":null,"discovery_flag":null,"number":12893,"object_type":"M","observation_date":"2003 12 01.30005","observatory_code":"704","original_record":"12893         C2003 12 01.30005 04 08 44.90 +17 46 55.2          18.3  cj3051704"},{"designation":null,"discovery_flag":null,"number":12893,"object_type":"M","observation_date":"2003 12 01.35827","observatory_code":"691","original_record":"12893         C2003 12 01.35827 04 08 41.59 +17 46 45.4          17.7 Vcj3051691"},{"designation":null,"discovery_flag":null,"number":12893,"object_type":"M","observation_date":"2003 12 01.38631","observatory_code":"691","original_record":"12893         C2003 12 01.38631 04 08 40.01 +17 46 40.6          17.8 Vcj3051691"},{"designation":null,"discovery_flag":null,"number":12893,"object_type":"M","observation_date":"2003 12 01.41437","observatory_code":"691","original_record":"12893         C2003 12 01.41437 04 08 38.43 +17 46 35.9          17.7 Vcj3051691"},{"designation":null,"discovery_flag":null,"number":12893,"object_type":"M","observation_date":"2003 12 04.28759","observatory_code":"703","original_record":"12893         C2003 12 04.28759 04 06 02.96 +17 38 54.5          17.6 Vdj3051703"},{"designation":null,"discovery_flag":null,"number":12893,"object_type":"M","observation
```

Comment:
@mommermi - This file needs to be added to the `paths_test` list in `mpc/setup_package.py`, that should fix the current test failure.

Reply:
Yes, good point. I added it to the list. Thanks for pointing this out!